scripts/bluesky-0MQ-proxy

```diff
@@ -1,5 +1,40 @@
 #!/usr/bin/env python3
+import threading
 import argparse
+import logging
+import sys
+
+from bluesky.callbacks.zmq import RemoteDispatcher
+
+def start_dispatcher(logfile):
+    """The dispatcher function
+    Parameters
+    ----------
+    logfile : string
+        string come from user command. ex --logfile=temp.log
+        logfile will be "temp.log". logfile could be empty.
+    """
+    d = RemoteDispatcher('localhost:'+str(out_port))
```

Comment:
Another minor thing: it's more readable to write this as `f'localhost:{out_port}'`. That employs a feature only available in Python 3.6 or higher, but since ophyd just dropped Python 3.5 support, I see no reason that bluesky can't do the same.

Reply:
I see